fontforge/cvundoes.c

```diff
@@ -3741,7 +3741,7 @@ return;
     /*  contain references, and the width of the original glyph is the same as*/
     /*  the width of the original referred character, then we should make sure */
     /*  that the new width of the glyph is the same as the current width of   */
-    /*  the referred char. We can't do this earlier because of foreward refs.  */
+    /*  the referred char. We can't do this earlier because of foreword refs.  */
```

Comment:
The original was lexically ambiguous, but it ought to be "forward".

Reply:
Fixed in  9f0a6b7